payment/src/main/java/org/killbill/billing/payment/logging/PaymentLoggingHelper.java

```diff
@@ -153,6 +159,17 @@ public static void logAPICallInternal(final Logger log,
                        .append(JOINER.join(paymentControlPluginNames))
                        .append("'");
             }
+            if (exception != null) {
+                final ErrorCode error = ErrorCode.fromCode(exception.getCode());
```

Comment:
You mean `getCode`? The attribute is a primitive `int`, so we should be safe.
Also, the `fromCode` method never returns `null`, it falls back to `__UNKNOWN_ERROR_CODE`.